shared/teams/team/container.js

```diff
@@ -148,6 +148,26 @@ const isExplicitAdmin = (memberInfo: I.Set<Constants.MemberInfo>, user: string):
   return info.type === 'owner' || info.type === 'admin'
 }
 
+const getOrderedMemberArray = (
+  memberInfo: I.Set<Constants.MemberInfo>,
+  // $FlowIssue you is a string
```

Comment:
Ah, I don't think this is a $FlowIssue -- `you` is passed in as a `?string`, so it suffices to type `you: ?string` instead of `you: string`, since you do check it for null/empty string below anyway.